### PR TITLE
feat(curriculum): add animations to block 4.2 of the English Curriculum

### PR DESCRIPTION
**Pull request by DanielRosa74**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
Looks like the tests need to be updated to accept the new scenes.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-popular-trends-in-technology/661eb97a2eb89a1ecd9a507a.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Listen to the audio and fill in the blank.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Tom: Is it safe?`
+
+`It's _ secure, but like any technology, it has its challenges and privacy concerns.`
```
Comment (moT01) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-discuss-popular-trends-in-technology/661eb97a2eb89a1ecd9a507a.md:23]:
```suggestion
`Alice: It's _ secure, but like any technology, it has its challenges and privacy concerns.`
```

**Review by moT01 — COMMENTED** [review]
This is looking good @DanielRosa74. The playback seems pretty consistent. I am still getting quite a few steps where the audio gets cut off a little early. Here's a list of the task numbers:

15-19,  23-24, 26, 30, 36-37, 39-43, 46-47, 50-53, 58, 68-70, 72-75, 94, 104-105

I used Chrome for testing. Do you want to take a quick look at some of those and see if you are seeing the same thing? And maybe make some adjustments. They could be off a little from adjustments you made because of the inconsistencies that were present from before.

**Comment by DanielRosa74** [comment]
> This is looking good @DanielRosa74. The playback seems pretty consistent. I am still getting quite a few steps where the audio gets cut off a little early. Here's a list of the task numbers:
> 
> 15-19,  23-24, 26, 30, 36-37, 39-43, 46-47, 50-53, 58, 68-70, 72-75, 94, 104-105
> 
> I used Chrome for testing. Do you want to take a quick look at some of those and see if you are seeing the same thing? And maybe make some adjustments. They could be off a little from adjustments you made because of the inconsistencies that were present from before.

Changes made. The sentences were cutting off in those places indicated. This should fix them.

**Comment by DanielRosa74** [comment]
> Looks like the tests need to be updated to accept the new scenes.

Solved.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**